software/contrib/polyrhythmic_sequencer.py

```diff
@@ -91,7 +93,7 @@ def _pitch_cv(self, note: str) -> float:
 
     def _set_pitch(self):
         pitch = self._pitch_cv(self.current_note())
-        self.pitch_cv.voltage(pitch)
+        self.pitch_cv.voltage(min(pitch,self.max_voltage))
```

Comment:
[discussion] I don't feel like the changes in this file are in the same spirit as the others in this PR. They're not replacing a hardcoded voltage with a configured one. 

I have similar concerns regarding the changes to quantizer.py.

Reply:
I agree with the above; I would revert this change. The goal of this MR was specifically to address hard-coded gate/trigger voltages. Anything that's dynamically calculated I intentionally left alone in the initial set of changes.

Reply:
The `.voltage()` function already clamps the output, so I've reverted this.